code/modules/mob/living/carbon/xenomorph/saddles.dm

```diff
@@ -0,0 +1,102 @@
+/obj/item/storage/backpack/marine/duffelbag/xenosaddle
+	name = "\improper Runner Saddle"
+	desc = "A rugged saddle, designed to be worn by runners. You can alt-click to change the style"
+	icon = 'icons/Xeno/saddles/saddles.dmi'
+	//Same as icon state; We need to cache it so that update icons works later in the code and fetches the right style
+	var/style = "cowboybags"
+	icon_state = "cowboybags"
+	worn_icon_state = null
+	storage_type = /datum/storage/backpack/duffelbag/saddle
+	///list of selectable styles & the actual name in the DMI file so the user doesn't see the codenames
+	var/list/style_list = list(
+		"Cowboy (Default)" = "cowboybags",
+		"Rugged" = "saddlebags",
+	)
+	///this is really janky but I need the inverted list to be able to display the player-facing name to the player using the codename as a key (its all shitcode, all the way down)
```

Comment:
I reaaaally don't like this, but it can be addressed in a seperate pr since multiple things have this type of issue

Reply:
Yeah, i hate it just as much but i can't think of a better way to do it off the top of my head.